app/src/main/java/com/mirandadayadkins/junglee/LoginActivity.kt

```diff
@@ -0,0 +1,69 @@
+package com.mirandadayadkins.junglee
+
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.widget.EditText
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+
+import com.mirandadayadkins.junglee.databinding.ActivityLoginBinding;
+
+class LoginActivity : AppCompatActivity() {
+
+    // Set up ViewBinding for the layout
+    private lateinit var binding: ActivityLoginBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        // Setting up binding instance and view instances
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        val view = binding.root
+        val logInButton = binding.logInButton
+        val signUpButton = binding.signUpButton
+
+        // Setting onClickListener for Log In Button - opens a dialog box to enter login credentials
+        logInButton.setOnClickListener {
+            showDialogBox()
+        }
+
+        // Setting onClickListener for Sign Up Button - opens New Account Activity
+        signUpButton.setOnClickListener {
+            val newAccountIntent = Intent(this, NewAccountActivity::class.java)
+            startActivity(newAccountIntent)
+        }
+
+        setContentView(view)
+    }
+
+    /*
+    // Helper functions:
+    */
+
+    // Opens dialog box for user to enter login credentials
+    private fun showDialogBox() {
+        // Creating a Linear Layout containing EditText fields for entering username and password
+        val factory = LayoutInflater.from(this)
+        val credentialsView = factory.inflate(R.layout.dialog_login, null)
+        val emailField = credentialsView.findViewById<EditText>(R.id.emailEntryField)
+        val passwordField = credentialsView.findViewById<EditText>(R.id.passwordEntryField)
+
+        // Creating Alert Dialog using above layout
+        val dialogBuilder = AlertDialog.Builder(this, R.style.AlertDialogCustom)
```

Comment:
Should probably be in its own class

Reply:
I don't see how this would be helpful, especially since I only have one alert dialog like this